src/sentry/static/sentry/app/actionCreators/streamTag.jsx

```diff
@@ -0,0 +1,29 @@
+import {Client} from '../api';
+import {t} from '../locale';
+import StreamTagStore from '../stores/streamTagStore';
+import StreamTagActions from '../actions/streamTagActions';
+import AlertActions from '../actions/alertActions';
+
+const api = new Client();
+
+const MAX_TAGS = 500;
+
+export function fetchStreamTags(orgId, projectId) {
```

Comment:
Tags aren't specific to stream right? Maybe remove "stream" references from the actions/store.

Reply:
done